lib/alephant/broker/response/factory.rb

```diff
@@ -4,12 +4,12 @@ module Alephant
   module Broker
     module Response
       class Factory
-        def self.response_for(request)
+        def self.response_for(request, env)
```

Comment:
I am passing the env into the response, so I can use the etag and last_modified values from the request. I could make env a property of request, and use this in the response instead, but thought it best to use env.